.github/workflows/images.yaml

```diff
@@ -0,0 +1,51 @@
+name: addon images
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+env:
+  IMAGE_REGISTRY: quay.io
+  IMAGE_ORGANIZATION: rhobs
+  IMAGE_ADDON_NAME: multicluster-observability-addon
+  IMAGE_VERSION: v0.0.1
+
+jobs:
+ publish-image:
+   runs-on: ubuntu-latest
+   steps:
+     - uses: actions/checkout@v3
+
+     - name: Set up QEMU
+       uses: docker/setup-qemu-action@v3
+
+     - name: Set up Docker Buildx
+       uses: docker/setup-buildx-action@v3
+
+     - name: Login to Quay.io
+       uses: docker/login-action@v3
+       with:
+         registry: quay.io
+         logout: true
+         username: ${{ secrets.RHOBS_USER }}
+         password: ${{ secrets.RHOBS_PASS }}
```

Comment:
@periklis I don't think I have permission to set this, I need your help in this step